docs/css/coding-guidelines.md

```diff
@@ -6,198 +6,230 @@ sidebar_label: Coding guidelines
 
 ---
 
-### Mobile-First Responsive Design
+### Space after function parameters
+
+Function parameters should have spaces after their separating commas, but not before:
 
-The mobile-first approach is a tenet of progressive enhancement. It is the ideology that mobile design, as the hardest, should be done first. Once the mobile design questions are answered, designing for other devices will be easier. What it boils down to is that, the smallest of the designs will have only the essential features.
+```css
+color: rgb(255, 0, 0);
+background-image: linear-gradient(to bottom, red, black);
+```
 
 ---
 
-### Basic breakpoints in Responsive Design
+### Syntax style
+
+There are a variety of CSS writing styles you can use, but we prefer the expanded style, with the selector/opening brace, close brace, and each declaration on a separate line. This maximizes readability, and again, promotes consistency on MDN Web Docs.
+
+In addition, keep these specifics in mind:
+
+- Include a space between the selector(s) and the opening curly brace.
+- Always include a semicolon at the end of the last declaration, even though it isn't strictly necessary.
+- Put the closing curly brace on a new line.
+- In each declaration, put a space after the separating colon, but not before.
+- Use two spaces for code indentation.
 
 ```css
-// Small devices (landscape phones, 576px and up)
-@media (min-width: 576px) {
-  ...;
+/* Good Approach */
+p {
+  color: white;
+  background-color: black;
+  padding: 1rem;
 }
+```
 
-// Medium devices (tablets, 768px and up)
-@media (min-width: 768px) {
-  ...;
-}
+```css
+/* Bad Approach */
+p { color: white; background-color: black; padding: 1rem; }
+```
 
-// Large devices (desktops, 992px and up)
-@media (min-width: 992px) {
-  ...;
-}
+---
 
-// Extra large devices (large desktops, 1200px and up)
-@media (min-width: 1200px) {
-  ...;
-}
+### Value to turn off properties
+
+When turning off borders (and any other properties that can take `0` or `none` as values), use `0` rather than `none`:
+
+```css
+border: 0;
 ```
 
-### Shrink CSS file size with CSS Compressors
+---
+
+### Shrinking CSS File Size using CSS Compressors
 
-It’s really a great idea to shrink the CSS file size as it will remove white spaces, line breaks and remove redundant CSS styles. Through this, you can help browsers to speed up the loading of your CSS codes. Using tools like CSS Compressor and CSS Compressor & Minifier can make this happen.
+If you feel that there is something wrong while the CSS codes get loaded over the browsers and it seems to be lagging behind in speed, then there is high time you tried to compress the size of the CSS files. A lot of elements, including line breaks, white spaces, and even redundant CSS styles might be interfering with your CSS file and delaying your site from loading quicker. Some of the tools that you can use to get rid of these issues include CSS Compressor or blend it with Minifier to utilize the benefits to its fullest.
```

Comment:
"If you measure that the CSS takes more time to download on an average network (throttle in network tab), then you may compress the CSS to shave off all the white spaces, new lines; event redundant styles."
"All these take extra time to download, parse and apply cascadingly."

Reply:
Done